restful.module

```diff
@@ -561,3 +564,18 @@ function restful_date_time_format_element_validate($element, &$form_state) {
     )));
   }
 }
+
+/**
+ * Implements hook_restful_resource_alter().
+ */
```

Comment:
``Decorate an existing resource with other services (e.g. rate limit and render cache).``